pkg/registry/service/portallocator/allocator.go

```diff
@@ -151,8 +151,7 @@ func (r *PortAllocator) Restore(pr util.PortRange, data []byte) error {
 	if !ok {
 		return fmt.Errorf("not a snapshottable allocator")
 	}
-	snapshottable.Restore(pr.String(), data)
-	return nil
+	return snapshottable.Restore(pr.String(), data)
```

Comment:
Ah - this is a different function than the others, so not a lost commit, just an error I made repeatedly in this file and failed to check for other instances of.  Thanks for catching!